README.md

```diff
@@ -1,312 +1,157 @@
-# ekotrace
+# `modality-probe`
 
-A distributed causal tracing system suitable for embedded use.
+`modality-probe` is an open source part of Auxon’s Modality™
+continuous verification & validation platform. Its role is to record
+events and track their causal relationships between the different
+parts of your system. Why? Because being able to stitch together a
+causal history of a system, particularly a system _under test_,
+provides a high-resolution lens into _what happened_. This history can
+then be used for testing, debugging, understanding emergent scenarios,
+and more.
 
-> **This work is currently in progress. We are planning to make
-> changes and will note [here](CHANGELOG.md) when you will need
-> to make an update.**
+While `modality-probe` is written in Rust, it targets C environments,
+particularly those of the embedded variety. The library used for
+recording events and exchanging causality data does not depend on any
+sort of standard library and could be used in bare-metal or RTOS
+environments.
 
-## Background
+## How do I use it?
 
-`ekotrace` tracks events of interest in a system as well as
-the causal relationships of those events.
+### Have Rust
 
-The trace data that `ekotrace` outputs shows, for example,
-whether event B happened because of event A, or
-whether event B happened independently of event A.
+Clone this repository, and start by navigating into its root folder.
+Then use [`rustup`](https://rustup.rs/), then you should have the
+cargo command available to you and the commands in the following
+section should work.
 
-Causal tracing works without relying on system clocks at all.
+### Build it
 
-`ekotrace` can trace one or many embedded components and provides
-information about the impact of each component on the system as
-a whole.
+#### Linux (Debian-flavored)
```

Comment:
We're about to remove Debian packaging in favor of tarballs in https://github.com/auxoncorp/modality-probe/pull/207.

Reply:
Maybe this is ok for now though, and we can update afterwards?

Reply:
I think I agree with @mullr, and that this should just come out.